scripts/new-version-pr-generator.js

```diff
@@ -0,0 +1,154 @@
+/* eslint-disable no-console */
+const {execSync} = require('child_process');
+const {readFileSync} = require('fs');
+const pathResolve = require('path').resolve;
+const {mkdir} = require('shelljs');
+const yaml = require('js-yaml');
+const semver = require('semver');
+
+const {version: PACKAGE_VERSION} = require('../package.json');
+const secrets = require('../secrets.json');
+const Retry = require('./utilities/retry');
+
+const repositories = ['polaris-styleguide', 'web'];
```

Comment:
Have we checked with the web foundations team that this is something they'd welcome?

Reply:
Great point, I can check with them. I think that our process is creating a PR in the web repository. 

So it would eliminate work for our team but might be frustrating for their team.

Reply:
I am going to remove it as they have to do a lot of tophatting for each PR so it would probably be more of an annoyance. It would be good to test this is working against another repo... Is there any you can think of that use `polaris/react`?